src/notifications/notificationsView.ts

```diff
@@ -9,35 +9,35 @@ import { dispose } from '../common/utils';
 import { NotificationSubjectType } from '../github/interface';
 import { IssueModel } from '../github/issueModel';
 import { PullRequestModel } from '../github/pullRequestModel';
-import { INotificationItem, LoadMoreNotificationsTreeItem, NotificationTreeDataItem } from './notificationItem';
-import { NotificationItem, NotificationsManager } from './notificationsManager';
-import { NotificationsProvider } from './notificationsProvider';
+import { isNotificationTreeItem, NotificationTreeDataItem, NotificationTreeItem } from './notificationItem';
+import { NotificationsManager } from './notificationsManager';
 
 export class NotificationsTreeData implements vscode.TreeDataProvider<NotificationTreeDataItem>, vscode.Disposable {
 	private readonly _disposables: vscode.Disposable[] = [];
 	private _onDidChangeTreeData: vscode.EventEmitter<NotificationTreeDataItem | undefined | void> = new vscode.EventEmitter<NotificationTreeDataItem | undefined | void>();
 	readonly onDidChangeTreeData: vscode.Event<NotificationTreeDataItem | undefined | void> = this._onDidChangeTreeData.event;
 
+	private _pageCount: number = 1;
 	private _computeNotifications: boolean = false;
 
-	constructor(private readonly _notificationsProvider: NotificationsProvider, private readonly _notificationsManager: NotificationsManager) {
+	constructor(private readonly _notificationsManager: NotificationsManager) {
 		this._disposables.push(this._onDidChangeTreeData);
-		this._disposables.push(this._notificationsManager.onDidChangeNotifications(updates => {
-			this._onDidChangeTreeData.fire(updates);
+		this._disposables.push(this._notificationsManager.onDidChangeNotifications(() => {
+			this._onDidChangeTreeData.fire();
```

Comment:
nit: now that we have `this.refresh(false)` it would be nice to call that instead of firing the event emitter directly.